src/main/java/MyStoreRegistration.java

```diff
@@ -0,0 +1,47 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class MyStoreRegistration {
+    public static void main(String[] args) {
+        WebDriver driver = new ChromeDriver();
+        driver.get("http://automationpractice.com/index.php");
+        driver.findElement(By.xpath("//*[@id=\"header\"]/div[2]/div/div/nav/div[1]/a")).click();
```

Comment:
this path is quite not stable. For example, if another div is added before `div[2]` the whole scenario will fail.
It would be better to use class names for anchors, like `header / nav / div class=header_user_info / a class=login`